AppCenterAnalytics/AppCenterAnalytics/Internals/Model/Properties/MSDoubleTypedProperty.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+
+#import "MSTypedProperty.h"
+
+@interface MSDoubleTypedProperty : MSTypedProperty
+
+/**
+ * Double property value. Saved as NSNumber
```

Comment:
Rephrase or add a `.` at the end, please.

Reply:
Yep, actually removed the last part before even seeing this feedback 👍